internal/resources/cloud/resource_cloud_stack_test.go

```diff
@@ -31,6 +31,7 @@ func TestResourceStack_Basic(t *testing.T) {
 		resource.TestMatchResourceAttr("grafana_cloud_stack.test", "id", common.IDRegexp),
 		resource.TestCheckResourceAttr("grafana_cloud_stack.test", "name", resourceName),
 		resource.TestCheckResourceAttr("grafana_cloud_stack.test", "slug", resourceName),
+		resource.TestCheckResourceAttr("grafana_cloud_stack.test", "description", stackDescription),
```

Comment:
This will need to be set in the test config

Reply:
Hey @julienduchesne, I have amended the test config as you rightly pointed out. However, I am struggling with the Cloud unit tests as I am getting the following `resource_cloud_stack_test.go:45: Step 2/7, expected an error but got none` but I also get this when running them against the master branch directly, so it doesn't appear new with my changes.

I'm not really sure if it is something to do with my org or perhaps my dev machine does not produce quite the same error to meet the regex

Reply:
Cloud tests are tricky to run. I'll try them out on our end

Reply:
That would be much appreciated, thank you. I'm happy to try to fix anything that comes out of them.